AutoMad/madDictionary.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Nov  6 02:36:40 2024
+
+@author: dragon
+"""
+
+from dataclasses import dataclass
+import numpy as np
+import pandas as pd
+import random
+import wordFormer
+
+@dataclass
+class madDictionary:
+    dictionary : pd.DataFrame
+    
+    def getWord(self, wordType):
+        for category in self.dictionary.columns:
+            if wordType == category:
+            #print(random.choice(df[category].dropna()))
+                return random.choice(list(self.dictionary[category].dropna()))
```

Comment:
You use `list(self.dictionary[category].dropna())` several times, it might be worth creating a helper method for that